components/aks/nsg.tf

```diff
@@ -0,0 +1,38 @@
+data "azurerm_resources" "aks_nsg" {
+  count = var.cluster_count
+  resource_group_name = format("%s-%s-%s-aks-node-rg",
+    var.project,
+    var.environment,
+    "0${count.index}"
+  )
+  type = "Microsoft.Network/networkSecurityGroups"
+}
+resource "azurerm_network_security_rule" "AllowInternetToOAuthProxy" {
+  count                       = contains(["demo"], var.environment) ? 1 : 0
+  name                        = "AllowInternetToOAuthProxy"
+  priority                    = 100
+  direction                   = "Inbound"
+  access                      = "Allow"
+  protocol                    = "Tcp"
+  source_port_range           = "*"
+  destination_port_range      = "80, 443"
+  source_address_prefix       = "*"
+  destination_address_prefix  = "51.11.25.221, 20.68.184.102"
```

Comment:
These IPs correct with the new PIPs created?

Reply:
not created any new ones as re-used the existing ones.